robottelo/ui/locators/menu.py

```diff
@@ -5,10 +5,8 @@
 from .model import LocatorDict
 
 NAVBAR_PATH = (
-    '//div[contains(@class,"navbar") '
-    'and contains(@class,"persist-header") and '
-    'not(contains(@style, "display: none")) and '
-    'not(contains(@style, "display:none"))]'
+    '//div[contains(@class,"navbar-inner") and '
```

Comment:
That locator doesn't work for all situations and sometimes after some actions it starts to point to two elements instead one and as a consequence I got a fail for bunch of my test cases. That new one works fine as you can see in my test results for this PR.
Verified both for downstream and upstream builds